src/app/domains/Form/components/FormSimpleView/FormSimpleView.js

```diff
@@ -47,7 +47,12 @@ function FormSimpleView(props) {
       bodyStyle={styles.cardBodyPadding}
       cover={
         <Image
-          style={styles.cardImageStyles}
+          style={{
```

Comment:
move styles to separated file

Reply:
for example 
styles = {
   cardSizeSmall: {
      height: '90px',
      width: '140px'
   },
   cardSizeLarge: {
      height: '140px',
      width: '200px'
   },
   ...
}

Reply:
okay